docs: add slack username helptext

R-HNF opened the pull request:
⚠️⚠️⚠️ Since we do not accept all types of pull requests and do not want to waste your time. Please be sure that you have read pull request rules:
https://github.com/louislam/uptime-kuma/blob/master/CONTRIBUTING.md#can-i-create-a-pull-request-for-uptime-kuma

Tick the checkbox if you understand [x]:
- [x] I have read and understand the pull request rules.

# Description

Add help text about how to use the slack `Username` and the difference from mentions.

Fixes #4560 

## Type of change

Please delete any options that are not relevant.

- User interface (UI)

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I ran ESLint and other linters for modified files
- [x] I have performed a self-review of my own code and tested it
- [x] I have commented my code, particularly in hard-to-understand areas (including JSDoc for methods)
- [x] My changes generates no new warnings
- [ ] My code needed automated testing. I have added them (this is optional task)

## Screenshots (if any)

Please do not use any external image service. Instead, just paste in or drag and drop the image here, and it will be uploaded automatically.

<img width="527" alt="スクリーンショット 2024-10-28 9 55 45" src="https://github.com/user-attachments/assets/c3ba5e03-81fe-45b7-b3e5-651a62fd513e">


My slack test message:
<img width="295" alt="image" src="https://github.com/user-attachments/assets/d6a5dd06-1a70-4dca-8c46-7394d6ac353a">

CommanderStorm commented on src/components/notifications/Slack.vue:
```diff
@@ -20,6 +20,9 @@
                 <a href="https://api.slack.com/messaging/webhooks" target="_blank">https://api.slack.com/messaging/webhooks</a>
             </i18n-t>
+            <p style="margin-top: 8px;">
+                {{ $t("aboutUsername", [$t("slack"), $t("Username"), $t("Friendly Name")]) }}
```
I know that in other translations (in this file) this is done, but I don't like this pulling things out unless that makes translation simpler.

=> let's inline this one

```suggestion
                {{ $t("aboutSlackUsername") }}
```

R-HNF replied:
Indeed, it seems easier to manage.

(Also, when viewed in Japanese, it feels somewhat difficult to read since only parts are translated.)

CommanderStorm commented on src/lang/en.json:
```diff
@@ -731,6 +731,7 @@
     "Icon Emoji": "Icon Emoji",
     "signalImportant": "IMPORTANT: You cannot mix groups and numbers in recipients!",
     "aboutWebhooks": "More info about Webhooks on: {0}",
+    "aboutUsername": "The {0} {1} can change the display name of the message sender. This is not for mentions. If you want to mention someone, include it in the {2}.",
```
lets reword it like this, what do you think?

```suggestion
    "aboutSlackUsername": "Changes the display name of the message sender. If you want to mention someone, include it in the friendly name instead.",
```

CommanderStorm reviewed (commented):
What do you think about rewording it like this?

CommanderStorm commented on src/components/notifications/Slack.vue:
```diff
@@ -20,6 +20,9 @@
                 <a href="https://api.slack.com/messaging/webhooks" target="_blank">https://api.slack.com/messaging/webhooks</a>
             </i18n-t>
+            <p style="margin-top: 8px;">
+                {{ $t("aboutUsername", [$t("slack"), $t("Username"), $t("Friendly Name")]) }}
```
also, lets move this up to where the username field is, so that it is not so out of place

R-HNF commented:
@CommanderStorm 
Thank you for your comment! I addressed it in this commit 527d40be2dbd58aeab79e5e73dd6d5df38e3c0e6, also updated the screenshot in the PR description.

I feel that the helptext has become clearer. Thank you!

CommanderStorm commented on src/components/notifications/Slack.vue:
```diff
@@ -4,6 +4,11 @@
         <label for="slack-username" class="form-label">{{ $t("Username") }}</label>
         <input id="slack-username" v-model="$parent.notification.slackusername" type="text" class="form-control">
+        <div class="form-text">
+            <p style="margin-top: 8px;">
+                {{ $t("aboutSlackUsername") }}
+            </p>
```
```suggestion
            {{ $t("aboutSlackUsername") }}
```

CommanderStorm reviewed (approved):
Thanks for the improvements to the docs ♥️
